stack.py

```diff
@@ -102,5 +115,22 @@ def eval_program(program):
     return stack
 
 
+def try_float(string):
+    try:
+        return float(string)
+    except ValueError:
+        return string
+
+
+def get_argument(instuction, default):
```

Comment:
`get_argument` isn't being used for anything right now, and since it's doing `instruction[1]` it's definitely incorrect. I recommend removing and squashing with whichever commit added it.